tests/functional/tokenstorage_v2/test_json_token_storage.py

```diff
@@ -11,25 +11,25 @@
 
 
 @pytest.fixture
-def filename(tempdir):
-    return os.path.join(tempdir, "mydata.json")
+def json_file(tmp_path):
+    return tmp_path / "mydata.json"
 
 
-def test_file_does_not_exist(filename):
-    adapter = JSONTokenStorage(filename)
+def test_file_does_not_exist(json_file):
+    adapter = JSONTokenStorage(json_file)
     assert not adapter.file_exists()
 
 
-def test_file_exists(filename):
-    open(filename, "w").close()  # open and close to touch
-    adapter = JSONTokenStorage(filename)
+def test_file_exists(json_file):
+    open(json_file, "w").close()  # open and close to touch
```

Comment:
As above: is this actually testing anything interesting?

```suggestion
    json_file.touch()
```

Reply:
As above, maybe not! But let's leave it in place for the moment.